snomed-database-loader/create-database-postgres.sql

```diff
@@ -70,8 +70,9 @@ WITH RECURSIVE immunization_heirarchy AS (
              JOIN immunization_heirarchy i ON r.destinationId = i.conceptId
     WHERE r.typeId = '116680003' -- relationshipType (typeId) is 'IsA' (child of)
 )
-SELECT conceptId
-FROM immunization_heirarchy;
+SELECT ih.conceptId, ds.id as descriptionid
+FROM immunization_heirarchy ih
+JOIN snomedct.description_s ds on ih.conceptId = ds.conceptid;
 
 CREATE INDEX immunization_codes_conceptid_idx ON immunization_codes
     USING btree (conceptid);
```

Comment:
Sorry for the randomly timed comment, but it popped into my head. This `btree` won't be as effective as it was previously now that the SELECT query we perform needs to perform a search across two columns.

If we were able to combine the `conceptId` and `descriptionId` into one column, and have a single btree for both, I suspect that would be the most performant solution.

Reply:
I agree, great spot! Will address this ASAP.